docs/counter.md

```diff
@@ -115,6 +100,21 @@ readable decimal number, though, so we apply the scale to format it. This
 formatted version of the amount gets written to the `total` span on the page.
 
 ```js
-  const formatted = (total * Math.pow(10, -scale)).toFixed(scale)
-  document.getElementById('total').innerText = formatted
+const formatted = (total * Math.pow(10, -scale)).toFixed(scale)
+document.getElementById('total').innerText = formatted
 ```
+
+## Interactive example
+
+Click the **View as Web Monetized/non-Web Monetized visitor** button to see how much you've streamed.
```

Comment:
Could we add a note here along the lines of:

"This example simulates what a Web Monetized viewer would see and does not require you to have Web Monetization enabled in your own browser. No real payment is occuring"

I think a disclaimer along those lines would be good for all the examples

Reply:
Added disclaimer text to each example.